modules/programs/zsh.nix

```diff
@@ -529,7 +529,7 @@ in
         . "${config.home.profileDirectory}/etc/profile.d/hm-session-vars.sh"
 
         # Only source this once
-        if [[ -z "$__HM_ZSH_SESS_VARS_SOURCED" ]]; then
+        if [[ -z "''${__HM_ZSH_SESS_VARS_SOURCED-}" ]]; then
```

Comment:
I usually see a column before the dash https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash . Is that syntax official ? or zsh specific maybe

Reply:
This syntax is universal to POSIX-compliant shells, and in this situation makes slightly more sense semantically. See [the table here](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) for a comparison between them (scroll a bit down).